### ISSUE TITLE
Definitir dados da vaga

### ISSUE THREAD

**Issue by mpinheir** [OsProgramadores/portalestagio#14]
Objetivo:

Definiri quais os dados da vaga de estágio que serão capturados e armazenados pelo sistema

**Comment by lucasfturos** [comment]
Alguns dados importantes para coletar para as vagas seriam:
- Nome da vaga;
- Descrição da vaga;
- Tipo de contratação: CLT ou PJ;
- Número de vagas disponíveis pela empresa;
- Tipo da vaga: Tempo integral, meio período, estágio e etc.;
- Se a vaga é remota, híbrida ou presencial. 

Depois a gente pode colocar mais coisas, bom que vai ajudar a procurar as vagas pelos filtros do site (ideia de feature na busca).

**Comment by magnussolidus** [comment]
Visto que o portal é um portal para vagas de Estágio, não vejo necessidade para adicionar tipo de contratação. Estágio tem legislação própria e não deve ser tratado como emprego, quer seja CLT ou PJ.

Aqui minha sugestão de dados para as vagas:

- [x] Id - Alguma forma de identificador único para a vaga
- [x] Empresa do Local do Estágio (Não devem ser os dados da intermediária, quando houver)
- [x] Modalidade do Estágio (Presencial, Remoto, Híbrido, Outro)
- [x] Carga Horária 
- [x] Remuneração - Estágio pode ser não remunerado ou ter bolsa-auxílio. 
- [x] Benefícios
- [x] Responsabilidades
- [x] Responsável pelo Estágio - Pessoa responsável pelo estagiário e/ou informações sobre quem será responsável por lhe ensinar as coisas que você deve aprender no estágio.
- [x] Data de Inserção - Quando a vaga foi inserida na plataforma
- [x] Data de publicação - Quando a vaga foi publicada na plataforma
- [x] Data de Expiração - Quando a vaga deve ser encerrada automaticamente/quando foi fechada.

**Comment by Zezola** [comment]
Concordo, como é para estágio, não precisa da opção de ser CLT ou PJ.
Acho que podemos ter também: 

*CNPJ da empresa
*Cursos desejados
*Status (preenchida ou aberta)
*Quantidade de vagas

**Comment by mpinheir** [comment]
CNPJ é dado público da empresa?

**Comment by lucasfturos** [comment]
Sim, o CNPJ é dado público

**Comment by erickofs** [comment]
Se a vaga não estiver mais aberta, acredito que convém não exibir em pesquisa, já que não faz sentido mostrar pra uma pessoa uma vaga que ela não pode se candidatar. Logo, status de aberto ou fechado pode ficar irrelevante ou pode manter apenas como dado interno do portal pra qualquer análise.

**Comment by mpinheir** [comment]
Erick. Sim, vaga fechada não deve ser exibida na pesquisa mas o dado sec está fechada ou não precisa ser mantido no banco pois senão não vais ter como saber se a vaga já foi preenchida.

**Comment by erickofs** [comment]
https://www.bbc.com/portuguese/articles/crg4x4rxe91o

Esse é um ótimo exemplo de melhoria a fazer:

Se a vaga estiver publicada há mais de 90 dias no contador interno, ela também poderia deixar de ser exibida em pesquisa de usuário, mas ficaria ativa na plataforma, caso a empresa queira usar.

Exemplo: Existem programas de contratação, tipo o GenO da Oracle ou o Universia do Santander.

Eles publicam a vaga do programa e ela fica lá por tempo indeterminado. 

Assim, nós períodos de recrutamento os recrutadores conseguem direcionar candidatos mas essas vagas não aparecem na pesquisa ativa

**Comment by Mouraman** [comment]
Se querem manter isto para lingua portugusa e com oportunidades nos paises da mesma, recomendo que os dados sejam algo como :
- nome da empresa contratante;
- vaga a ser preenchida;
-faixa salarial;
-regime de trabalho;
-linguagens/frameworks necessarias;
- experiencia previa necessaria;
- requerimentos que adicionam a candidatura (conhecimentos que sao valorizados);
